test/interop/C/llvm/makefiles/commandLineLib.future

```diff
@@ -1 +0,0 @@
-bug: support --library-makefile with LLVM
```

Comment:
This test has a .bad file that should also get deleted with the .future file

Reply:
Great thanks.